client/src/auth/saga.js

```diff
@@ -15,11 +17,11 @@ export function* handleLogin({ payload: { history } }) {
     const res = yield axios.get(`/player/${currentUser.id}`);
     const { id, name } = res.data.player;
     yield put(authActions.loginSuccess({ id, name }));
-    yield put(alertActions.addAlert('Login successful!', 'success'));
+    yield put(homeActions.refreshActiveGames({ id: currentUser.id }));
```

Comment:
Instead of triggering this action here, we probably want to use a `useEffect` hook in our `Home` container.